### DISCUSSION TITLE
Meta reducers ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#28] [Announcements]
# Meta reducers ⚡️ Angular с примерами кода

NgRx Meta Reducers это функции, вызываемые сразу после возникновения действия, но перед тем, как они будут переданы глобальному редюсеру

https://angdev.ru/ngrx/meta-reducers/

**Comment by KupnoH** [comment]
Похоже ошибка в app.module.ts в импорте
Должно быть так:
```
@NgModule({
  imports: [
    StoreModule.forRoot(reducers, { metaReducers: [log] }),
  ],
  // ...
})
...
```
Из того, что вычитал в интернете `metaReducers` передаются во второй параметр метода `forRoot()`.

Спасибо за статью
